src/main/java/de/muenchen/isi/api/controller/AbfrageController.java

```diff
@@ -88,7 +89,7 @@ public ResponseEntity<InfrastrukturabfrageDto> getInfrastrukturabfrageById(@Path
             @ApiResponse(responseCode = "400", description = "BAD_REQUEST -> Abfrage konnte nicht erstellt werden, überprüfen sie die Eingabe.", content = @Content(schema = @Schema(implementation = InformationResponseDto.class)))
     })
     @PreAuthorize("hasAuthority(T(de.muenchen.isi.security.AuthoritiesEnum).ISI_BACKEND_WRITE_ABFRAGE.name())")
-    public ResponseEntity<InfrastrukturabfrageDto> createInfrastrukturabfrage(@RequestBody @Valid @NotNull final InfrastrukturabfrageDto abfrageDto) throws EntityNotFoundException {
+    public ResponseEntity<InfrastrukturabfrageDto> createInfrastrukturabfrage(@RequestBody @Valid @NotNull @IsGeschossflaecheWohnenSobonUrsaechlichRequired final InfrastrukturabfrageDto abfrageDto) throws EntityNotFoundException {
```

Comment:
Bei der Annotationsbenennung von `IsGeschossflaecheWohnenSobonUrsaechlichRequired` der der Bestandteil `Is` weggelassen werden, da es sich um keine Logik handelt welche in true oder false zurückgibt.

z.B. anstelle `@IsGeschossflaecheWohnenSobonUrsaechlichRequired` -> `@GeschossflaecheWohnenSobonUrsaechlichRequired`

Reply:
erledigt